packages/graphql-server/src/ee/app.ts

```diff
@@ -399,7 +405,7 @@ export const createApp = async (): Promise<{app: Koa, server: ApolloServer, conf
           kcAdminClient.setAccessToken(accessToken);
           getInstanceType = instCache.get;
           getImage = imageCache.get;
-          readOnly = true;
+          readOnly = false;
```

Comment:
1. Could we remove all readOnly code for both `app.ts` and `ee/app.ts`
2. Remove readOnlyMiddleware code.